dts/bindings/gpio/ti,sn74hc595.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022, Matthias Freese
+# SPDX-License-Identifier: Apache-2.0
+
+description: SN74HC595 SPI based GPIO expander
+
+compatible: "ti,sn74hc595"
+
+include: [gpio-controller.yaml, spi-device.yaml]
+
+properties:
+    label:
+      required: true
+
+    rstb-gpios:
+      type: phandle-array
+      required: true
+      description: Reset pin
+
+    cs-gpios:
+      type: phandle-array
+      required: true
+      description: Chip select
```

Comment:
This looks incorrect to me. Since you are including spi-device.yaml, the chip select pin should be set on the bus, not the device itself. What am I missing?

Reply:
Nothing - you are correct ;). We updated the files - please have a look. Thanks for your support.